web/web.go

```diff
@@ -461,7 +459,12 @@ func setParentTable(w http.ResponseWriter, r *http.Request) {
 	tableInterleaveIssues := parentTableHelper(table, update)
 	updateSessionFile()
 	w.WriteHeader(http.StatusOK)
-	json.NewEncoder(w).Encode(tableInterleaveIssues)
+
+	if update {
+		json.NewEncoder(w).Encode(sessionState.conv)
+	} else {
+		json.NewEncoder(w).Encode(tableInterleaveIssues)
+	}
```

Comment:
We should update the comment at the top of this function as following to reflect this change:

// setParentTable checks whether specified table can be interleaved, and updates the schema to convert foreign // key to interleaved table if 'update' parameter is set to true. If 'update' parameter is set to false, then return        
// whether the foreign key can be converted to interleave table without updating the schema.

Reply:
ok